website/templates/util/render_node.mako

```diff
@@ -78,7 +78,7 @@
                     <dt><span class="date log-date" data-bind="text: log.date.local, tooltip: {title: log.date.utc}"></span></dt>
                     <dd class="log-content">
                         <span data-bind="if:log.anonymous">
-                            <span><em>A user</em></span>
+                           <span data-bind="text: anonymousUserName"></span>
```

Comment:
I'm not sure this works because we are in the context of a `foreach` loop. @haoyuchen1992 Have you tested this with anonymized VOLs?